cluster-stamp.bicep

```diff
@@ -109,6 +104,36 @@ resource clusterReaderRole 'Microsoft.Authorization/roleDefinitions@2018-01-01-p
   scope: subscription()
 }
 
+// Built-in Azure RBAC role that can be applied to a cluster or a namespace to grant read privileges to that scope for a user or group
```

Comment:
Copy+Paste for all the comments for these newly converted ones, but didn't get them updated to match what they actually are.  Can you do a quick sweep through and update the comments to match the role definition?

Reply:
good catch @ckittel, don't hesitate in proposing wording changes as they are more than welcome. 

done | addressed from e465dbcca392bd8abd68db9bc67ac00c3dbff46b

Reply:
Marking this one resolved, but I did add some wording suggestions in a comment below, which included a couple typo fixes.